controllers/nodeagent.go

```diff
@@ -42,9 +42,6 @@ const (
 )
 
 var (
-	fsPvHostPath    = getFsPvHostPath("")
-	pluginsHostPath = getPluginsHostPath("")
-
```

Comment:
@shubham-pampattiwar can you review if this is ok? you recently modified this in [here](https://github.com/openshift/oadp-operator/pull/1482/files#diff-e1f5e14f93b9b4945243094dc61c61c33665e3dc71897269e7c129ac7c6654ecR45-R46)

Reply:
yeah we need those

Reply:
remove usage in my changes